src/main.rs

```diff
@@ -73,15 +89,9 @@ fn main() -> Result<(), RunError> {
         .get_matches();
 
     let args = args.subcommand_matches("tarpaulin").unwrap_or(&args);
+
+    set_up_logging(args.is_present("verbose"));
+
     let config = Config::from(args);
-    let res = run(&config);
-    println!("Tarpaulin finished");
-    res
-    /*match run(&config) {
-        Ok(()) => println!("Tarpaulin finished"),
-        Err(e) => {
-            println!("Error during run");
-            std::process::exit(e);
-        },
-    }*/
+    run(&config).map_err(|e| error!("{}", e.to_string()));
```

Comment:
This prints errors in a pretty manner:

![screenshot from 2018-12-08 18-48-54](https://user-images.githubusercontent.com/8766773/49688949-00721c80-fb1a-11e8-9e2f-36d0932e2c84.png)


However, we still have to switch println!s to `info!` or the likes.